com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -21,19 +21,122 @@ public enum NetworkAuthority
             Shared
         }
 
-        private class NetworkState : INetworkSerializable
+        internal class NetworkState : INetworkSerializable
         {
-            public bool InLocalSpace;
-            public Vector3 Position;
-            public Quaternion Rotation;
-            public Vector3 Scale;
+            internal const int InLocalSpaceBit = 0;
+            internal const int PositionXBit = 1;
+            internal const int PositionYBit = 2;
+            internal const int PositionZBit = 3;
+            internal const int RotationXBit = 4;
+            internal const int RotationYBit = 5;
+            internal const int RotationZBit = 6;
+            internal const int ScaleXBit = 7;
+            internal const int ScaleYBit = 8;
+            internal const int ScaleZBit = 9;
+            // 10-15: <unused>
+            public ushort Bitset;
+
+            public bool InLocalSpace
+            {
+                get => (Bitset & (1 << InLocalSpaceBit)) != 0;
```

Comment:
I wish we could use Bitfield from collections for this https://docs.unity3d.com/Packages/com.unity.collections@0.4/api/Unity.Collections.BitField32.html

Reply:
haha, we could, only if we were to have `BitField16` :P